docs/components/MessagingAttachmentView.jsx

```diff
@@ -24,71 +24,6 @@ export default class MessagingAttachmentView extends React.PureComponent {
   static cssClass = cssClass;
```

Comment:
note: changes in this file are unrelated to the rest of the PR. Just some cleanup from recent changes to this component